edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Core/ConnectionReauthenticator.cs

```diff
@@ -1,28 +1,33 @@
 // Copyright (c) Microsoft. All rights reserved.
-
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
 namespace Microsoft.Azure.Devices.Edge.Hub.Core
 {
     using System;
     using System.Collections.Generic;
     using System.Linq;
     using System.Threading.Tasks;
     using System.Timers;
+
     using Microsoft.Azure.Devices.Edge.Hub.Core.Device;
     using Microsoft.Azure.Devices.Edge.Hub.Core.Identity;
     using Microsoft.Azure.Devices.Edge.Hub.Core.Identity.Service;
     using Microsoft.Azure.Devices.Edge.Util;
     using Microsoft.Extensions.Logging;
-    using Timer = System.Timers.Timer;
 
     public sealed class ConnectionReauthenticator : IDisposable
     {
-        readonly IConnectionManager connectionManager;
         readonly IAuthenticator authenticator;
+
+        readonly IConnectionManager connectionManager;
```

Comment:
Can we pls remove the rule that requires fields to be sorted alphabetically? I am not sure if it adds much value.

Reply:
Similar to method; I think sorting fields (properties and methods) increases readability. are you sure that you prefer to remove it?

Reply:
Updated format rule to just order fields by access.  And manually revert the order if it logical order looks better, let me know if I missed.